plugin/newebuild.vim

```diff
@@ -121,14 +121,17 @@ fun! <SID>MakeNewEbuild()
             " }}}
         elseif l:category ==# "dev-java"
             " {{{ dev-java generation-2 default java-pkg-simple ebuild
-            put ='JAVA_PKG_IUSE=\"doc source\"'
+            put ='JAVA_PKG_IUSE=\"doc source test\"'
             put ='MAVEN_ID=\"\"'
+            put ='MAVEN_PROVIDES=\"\"'
+            put ='JAVA_TESTING_FRAMEWORKS=\"junit-4\"'
             put =''
             put ='inherit java-pkg-2 java-pkg-simple'
             put =''
             put ='DESCRIPTION=\"\"'
             put ='HOMEPAGE=\"\"'
             put ='SRC_URI=\"\"'
+            put ='S=\"${WORKDIR}/${P}\"'
```

Comment:
I don't think that's a good idea. Since it's the same as default, it will immediately "flash red" and — in the worst case — people will leave it here because they would think it's necessary.

Reply:
> I don't think that's a good idea. Since it's the same as default,

`Default` in `java-pkg-simple.eclass` is [`S="${WORKDIR}"`](https://github.com/gentoo/gentoo/blob/master/eclass/java-pkg-simple.eclass#L34-L35).

Reply:
Ah, ok. I dare say that's a bad default but *shrug*.

Reply:
Fully agree, maybe something to adjust with `EAPI 9`?

Reply:
👍